.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
```

Comment:
I think this would be better handled by the PR labels system. We could agree on a set of labels that we provide (bug fix, enhancements, new features, compatibility break, etc)